lib/bindings/solo5_console_stubs.c

```diff
@@ -18,21 +18,18 @@
 
 #include "solo5.h"
 
-#include <string.h>
-
 #include <caml/alloc.h>
 #include <caml/memory.h>
 #include <caml/signals.h>
 #include <caml/fail.h>
 #include <caml/callback.h>
 #include <caml/bigarray.h>
 
-
-CAMLprim value stub_console_write(value arg) {
+CAMLprim value
+stub_console_write(value arg)
+{
     CAMLparam1(arg);
 
-    const char *str = String_val(arg);
-    solo5_console_write(str, strlen(str));
-
+    solo5_console_write(String_val(arg), caml_string_length(arg));
```

Comment:
while your PR didn't change anything, I want to raise the issue that `stub_console_write` ignores the returned number of bytes written here -- see https://github.com/mirage/mirage-flow/issues/4#issuecomment-306041566 for some more detail.  I suspect we want `stub_console_write` to return an `int`, and loop until all the bytes are written.